grammars/solution.yaml-tmLanguage

```diff
@@ -54,15 +54,15 @@ repository:
       - name: meta.generic.variable.other.solution
         match: '(?<![.|])([A-Za-z]+)\s*(=)'
         captures:
-          0: { name: variable.other.solution }
-          1: { name: punctuation.separator.declaration.solution }
+          a: { name: variable.other.solution }
+          b: { name: punctuation.separator.declaration.solution }
       - name: meta.generic.variable.property.solution
         match: '([A-Za-z]+)\s*(\|)'
         captures:
-          0: { name: variable.language.property.solution }
-          1: { name: punctuation.separator.property.solution }
+          a: { name: variable.language.property.solution }
+          b: { name: punctuation.separator.property.solution }
       - name: variable.other.member.solution
-        match: '(?<=[.|]\s*)[A-Za-z0-9 ]+'
+        match: '[.|]\s*[A-Za-z0-9 ]+'
```

Comment:
This should still be a look behind, just move the \s* outside.

Reply:
Ok. I had it with lookbehind with \s* outside earlier, but then thought LB is not necessary. Is there an example where it is needed?

Reply:
It's just to exclude the "|"/"." from being included in the match, as it should be marked as just punctuation and not be part of the member-variable token.